src/tools/mdbm_config.cc

```diff
@@ -42,7 +42,6 @@ static uint SYS_PAGESIZE;    // system page size
 static uint CONFIG_MAXPAGE = 1 << 24;   // 16MB (subtract system page size later)
 static const int MBYT = 1024 * 1024;
 static const uint ENTRY_OVERHEAD  =   8;     // MDBM per-entry overhead is 8 bytes
-static const uint PAGE_OVERHEAD   =  16;     // MDBM per-page overhead
```

Comment:
Yeah I dug pretty far back, and I didn't find any hints, so I just merged it.  :)
It might have been some carry-over from v2.
Thanks for the fix.